docs/reference.rst

```diff
@@ -5,12 +5,22 @@ Reference
 
     Get QR code from a gray scale image ('Y800' mode)
 
+    The image should only have one QR code.
```

Comment:
Since it is deprecated, I would mention `qr_code_scanner()` after `scan_codes()`. Also, perhaps we could add something like:

    If the image has more than one QR code, ``None`` is returned.

    This function is deprecated, use ``scan_codes()`` instead.